Python/tests/test.py

```diff
@@ -0,0 +1,32 @@
+from recognizers_text import Culture, ModelResult
+from recognizers_number import NumberRecognizer
+from recognizers_number_with_unit import NumberWithUnitRecognizer 
+from recognizers_date_time import DateTimeRecognizer 
+from recognizers_sequence import SequenceRecognizer 
+
+#import recognizers_suite
+
+
+#recognize_datetime("I'll go back 8pm today", Recognizers.Culture.English)
+
+inputStr = ["It was afternoon at 7"]#"Voy a volver dentro de 5 horas"]#, "I'll be out Tuesday from 2:00 to 2:30pm", "I'll be out from 5 to 6pm", "I'll be out between 5 and 6 in the afternoon"]#, "I am feeling sick before around 2pm today", "I am feeling sick after around 3:30pm", "I am feeling sick around 1pm"]
+
+recognizer = DateTimeRecognizer(Culture.English)
```

Comment:
Is this some local test code you use? There's a SimpleConsole project that can be used for testing too.

Reply:
But I don't think this file was supposed to be added to the repo, right?

Reply:
right, removed